src/templates/Topbar.vue

```diff
@@ -0,0 +1,39 @@
+<template>
```

Comment:
shall we use the template as is in phoenix? or shall we copy over the template?

Reply:
I recommend using as is. There will be no other places where this should or could be used.

Reply:
ok - so basically copying over the template section - right?